concent_api/core/views.py

```diff
@@ -1301,10 +1301,7 @@ def request_upload_status(file_transfer_token_from_database: message.concents.Fi
     }
     request_http_address = settings.STORAGE_CLUSTER_ADDRESS + CLUSTER_DOWNLOAD_PATH + file_transfer_token.files[0]['path']
 
-    cluster_storage_response = requests.head(
-        request_http_address,
-        headers = headers
-    )
+    cluster_storage_response = send_request_to_cluster_storage(headers, request_http_address)
```

Comment:
`cluster_storage` -> `storage_cluster`.

EDIT: I see that it was already called like this before your changes. But it's a trivial change. Could you fix it?

Reply:
done